profile_library/lidl/14158804L/model.json

```diff
@@ -0,0 +1,18 @@
+{
+    "calculation_strategy": "lut",
+    "created_at": "2024-10-14T11:07:29",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly Plug",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "master"
+    },
+    "aliases": [
+        "TS0505B"
+    ],
```

Comment:
This alias is too generic, this is a generic zigbee model description. There are hundreds of different light which match these. This must be removed.

```suggestion
```

Reply:
Alright :)